13-transport.Rmd

```diff
@@ -225,18 +226,18 @@ names(zones_joined)
 ```
 
 The result is `zones_joined`, which contains new columns representing the total number of trips originating in each zone in the study area (almost 1/4 of a million) and their mode of travel (by bicycle, foot, car and train).
-The geographic distribution of trip origins is illustrated in the left-hand map in Figure \@ref(fig:zones).
+The geographic distribution of trip origins is illustrated in the left-hand panel in Figure \@ref(fig:zones).
 This shows that most zones have between 0 and 4,000 trips originating from them in the study area.
 More trips are made by people living near the center of Bristol and fewer on the outskirts.
 Why is this?
 Remember that we are only dealing with trips within the study region: low trip numbers in the outskirts of the region can be explained by the fact that many people in these peripheral zones will travel to other regions outside of the study area.
-Trips outside the study region can be included in regional model by a special destination ID covering any trips that go to a zone not represented in the model [@hollander_transport_2016].
+Trips outside the study region can be included in a regional model by a special destination ID covering any trips that go to a zone not represented in the model [@hollander_transport_2016].
 The data in `bristol_od`, however, simply ignores such trips: it is an 'intra-zonal' model.
 
 In the same way that OD datasets can be aggregated to the zone of origin, they can also be aggregated to provide information about destination zones.
 People tend to gravitate towards central places.
 This explains why the spatial distribution represented in the right panel in Figure \@ref(fig:zones) is relatively uneven, with the most common destination zones concentrated in Bristol city center.
-The result is `zones_od`, which contains a new column reporting the number of trip destinations by any mode, is created as follows:
+The result is `zones_od`, which contains a new column reporting the number of trip destinations by any mode, and it iscreated as follows:
```

Comment:
```suggestion
The result is `zones_od`, which contains a new column reporting the number of trip destinations by any mode, and it is created as follows:
```

Reply:
Thanks, good fix. I need to improve my regex skills.